apps/remote_control/lib/lexical/remote_control/code_intelligence/entity.ex

```diff
@@ -93,13 +93,36 @@ defmodule Lexical.RemoteControl.CodeIntelligence.Entity do
          :struct <- kind_of_alias(path) do
       resolve_struct(charlist, node_range, document, position)
     else
-      _ -> resolve_module(charlist, node_range, document, position)
+      _ ->
+        if follow_percent?(document, position) do
```

Comment:
we can't fetch the path for some code like this:

```elixir
      code = ~q[
        defmodule MyModule do
          |%MyStruct{}
        end
      ]
```

not sure if this is a bug of `path_at`

Reply:
This does look like a bug in `path_at`. It should return the `%` node. I'll look into it.

Reply:
This should be fixed by:

- #419